Makefile

```diff
@@ -26,7 +26,8 @@ __check_defined = \
     $(if $(value $1),, \
       $(error Undefined $1$(if $2, ($2))))
 
-all: tools build test fmtcheck vet validate_commits image ## Compiles fabric8-jenkins-proxy and runs format and style checks
+.PHONY: all
+all: tools build test fmtcheck vet validate_commits image lint ## Compiles fabric8-jenkins-proxy and runs format and style checks
```

Comment:
Could you please move the `lint` target after vet. As it stands now it comes too late. Really once the image builds you want to have completed all checks